src/index.html

```diff
@@ -6,9 +6,94 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a href="index.html" class="logo">
+          <img src="images/logo.png"
+            alt="logo"
+            class="logo__img"
+          >
```

Comment:
```suggestion
          <img 
            src="images/logo.png"
            alt="logo"
            class="logo__img"
          >
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/7d38de65-c37a-4482-8a0a-038af0715625)